tests/model_bulk_create_test.js

```diff
@@ -0,0 +1,180 @@
+require('./helper');
+
+const { expect } = require('chai');
+const { Sequelize, DataTypes } = require('../source');
+
+describe('Model', () => {
+  beforeEach(async function() {
+    this.User = this.sequelize.define('User', {
+      username: DataTypes.STRING,
+      secretValue: {
+        type: DataTypes.STRING,
+        field: 'secret_value'
+      },
+      data: DataTypes.STRING,
+      intVal: DataTypes.INTEGER,
+      theDate: DataTypes.DATE,
+      aBool: DataTypes.BOOLEAN,
+      uniqueName: { type: DataTypes.STRING, unique: true }
+    });
+    this.Account = this.sequelize.define('Account', {
+      accountName: DataTypes.STRING
+    });
+    this.Student = this.sequelize.define('Student', {
+      no: { type: DataTypes.INTEGER, primaryKey: true },
+      name: { type: DataTypes.STRING, allowNull: false }
+    });
+    this.Car = this.sequelize.define('Car', {
+      plateNumber: {
+        type: DataTypes.STRING,
+        primaryKey: true,
+        field: 'plate_number'
+      },
+      color: {
+        type: DataTypes.TEXT
+      }
+    });
+
+    await this.sequelize.sync({ force: true });
+  });
+
+  describe('bulkCreate', () => {
+    it.skip('supports transactions', async function() {
+      const User = this.sequelize.define('User', {
+        username: DataTypes.STRING
+      });
+      await User.sync({ force: true });
+      const transaction = await this.sequelize.transaction();
+      await User.bulkCreate([{ username: 'foo' }, { username: 'bar' }], { transaction });
+      const count1 = await User.count();
+      const count2 = await User.count({ transaction });
+      expect(count1).to.equal(0);
+      expect(count2).to.equal(2);
+      await transaction.rollback();
+    });
+
+    // Reason: CRDB does not grant autoIncrement to be sequential and usually does not gives it
+    // small numbers like PG does. Reimplementing the test to check if it is incremental below.
+    describe('return values', () => {
+      it.skip('should make the auto incremented values available on the returned instances', async function() {
+        const User = this.sequelize.define('user', {});
+
+        await User
+          .sync({ force: true });
+
+        const users0 = await User.bulkCreate([
```

Comment:
Let's keep formatting consistent, it looks like you kept the array on the same line for the other definitions.

```const users = await User.bulkCreate([{}, {}, {}], { returning: true });```